i915.c

```diff
@@ -473,7 +473,8 @@ static int i915_add_combinations(struct driver *drv)
 		struct format_metadata metadata_y_tiled = { .tiling = I915_TILING_Y,
 							    .priority = 3,
 							    .modifier = I915_FORMAT_MOD_Y_TILED };
-		if (drv->gpu_grp_type & GPU_TYPE_DUAL_IGPU_DGPU) {
+		if ((drv->gpu_grp_type & GPU_GRP_TYPE_HAS_INTEL_DGPU_BIT) ||
+		    (drv->gpu_grp_type & GPU_GRP_TYPE_HAS_VIRTIO_GPU_BLOB_P2P_BIT)) {
```

Comment:
Why not code here like the following, i think this is more similar to the original code:
		if ((drv->gpu_grp_type & GPU_GRP_TYPE_HAS_INTEL_IGPU_BIT) &&
		    (drv->gpu_grp_type & GPU_GRP_TYPE_HAS_INTEL_DGPU_BIT)) {

Reply:
Arriving here means we are iGPU.

Condition `(drv->gpu_grp_type & GPU_GRP_TYPE_HAS_INTEL_DGPU_BIT)` implies there is also a dGPU (this covers the case of original code), and `(drv->gpu_grp_type & GPU_GRP_TYPE_HAS_VIRTIO_GPU_BLOB_P2P_BIT)` implies that we have a dGPU display.